polynote-frontend/polynote/ui/component/notebook/notebookconfig.ts

```diff
@@ -71,7 +72,8 @@ export class NotebookConfigEl extends Disposable {
                             const conf = new NotebookConfig(dependencies.conf, exclusions.conf, resolvers.conf, spark.conf, spark.template, kernel.envVars, kernel.scalaVersion, kernel.jvmArgs);
                             this.copyConfig(conf);
                         }),
-                        button([], {}, ['Paste & Save Configuration']).click(() => this.pasteConfig())
+                        button([], {}, ['Paste & Save Configuration']).click(() => this.pasteConfig()),
+                        this.pasteErrorMessage = para(['hide', 'error-message'], ['Paste failed - your clipboard does not contain valid JSON'])
```

Comment:
Hmm, is it bad for the error message to always be hanging out in the DOM?

Reply:
Probably not bad (since it's so small), but no reason for it to either. I'll change it to be added in/out of the DOM instead.